README.md

```diff
@@ -2,7 +2,7 @@
 [![Build Status](https://travis-ci.org/theia-ide/typescript-language-server.svg?branch=master)](https://travis-ci.org/theia-ide/typescript-language-server)
 
 # TypeScript Language Server
-[Language Server Protocol](https://github.com/Microsoft/language-server-protocol) implementation for TypeScript wrapping `tsserver`.
+[Language Server Protocol](https://github.com/Microsoft/language-server-protocol) implementation for TypeScript wrapping `tsserver`. You must install it first via `$ npm install -g typescript`.
```

Comment:
Not necessary, it can come with multiple places, i.e. your project can declare it as a dev dependency.

Reply:
I understand, yeah. I should just precise you need typescript in any way?

Reply:
I also think this shouldn't be added. Installing globally is [officially considered bad practice](https://nodejs.org/en/blog/npm/npm-1-0-global-vs-local-installation/#which-to-choose).

Instead, I'd suggest linking to the `tsserver` documentation, which has more info about how to get and use it.

```suggestion
[Language Server Protocol](https://github.com/Microsoft/language-server-protocol) implementation for TypeScript wrapping [`tsserver`](https://github.com/Microsoft/TypeScript/wiki/Standalone-Server-%28tsserver%29).
```